frontend/src/components/5.Lobby/Lobby.css

```diff
@@ -0,0 +1,110 @@
+
+/*Lobby box */
+.LobbyBox {
+    outline: none;
+    border: 4px solid #000000;
+    background-color: white;
+    height: 30vw;
+    width: 75vw;
+    font-family: Comic Sans MS;
+    border-radius: 30px;
+    padding: 2vw;
+    font-size: 3vw;
+    /* margin-top: 20vw; */
```

Comment:
Can remove comment if we don't need it anymore

Reply:
Removed the comment. Thanks, another good spotting!